A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_CRZ.html

```diff
@@ -0,0 +1,97 @@
+﻿
+<link rel="stylesheet" href="./A320_Neo_LowerECAM_CRZ.css" />
+
+<script type="text/html" id="LowerECAMCRZTemplate">
+<!--div id="a320-neo-lower-ecam-crz" style="width: 600px; height: 600px; background-color: black;"-->
```

Comment:
Clarify reason of commented code or remove it entirely

Reply:
Was just for my testing, so I could view it as a website in a browser. I will remove it later.